legal-api/src/legal_api/core/filing.py

```diff
@@ -110,6 +111,9 @@ class FilingTypesCompact(str, Enum):
         AMALGAMATION_APPLICATION_REGULAR = 'amalgamationApplication.regular'
         AMALGAMATION_APPLICATION_VERTICAL = 'amalgamationApplication.vertical'
         AMALGAMATION_APPLICATION_HORIZONTAL = 'amalgamationApplication.horizontal'
+        TRANSPARENCY_REGISTER_ANNUAL = 'transparencyRegister.ANNUAL'
+        TRANSPARENCY_REGISTER_CHANGE = 'transparencyRegister.CHANGE'
+        TRANSPARENCY_REGISTER_INITIAL = 'transparencyRegister.INITIAL'
```

Comment:
camelCase would be better to align with existing subtypes

Reply:
sure thing, will update